code/modules/jobs/job_types/ncr.dm

```diff
@@ -201,6 +201,43 @@ Medic
 
 
 
+/*
+Quartermaster
+*/
+/datum/job/ncr/f13quartermaster
+	title = "NCR Quartermaster"
+	flag = F13QUARTERMASTER
+	faction= "NCR"
+	total_positions = 1
+	spawn_positions = 1
+	forbids = "The NCR forbids: Drugs (except for med-X), Drinking, Gambling, and sex on duty."
+	enforces = "The NCR expects: Obeying the lawful orders of superiors. Proper treatment of prisoners.  Good conduct within the Republics laws. Wearing the uniform."
+	description = "You are in charge of requisitions and organizing the armory for the NCR camp. You answer directly to the Lieutenant and Captain, while also organizing grunts to assist you as needed."
+	supervisors = "Lieutenants and above"
+	selection_color = "fff5cc"
+	exp_requirements = 1200
+
+	outfit = /datum/outfit/job/ncr/f13quartermaster
+
+/datum/outfit/job/ncr/f13quartermaster
+	name = "NCR Quartermaster"
+	jobtype = /datum/job/ncr/f13quartermaster
+	id = 			/obj/item/card/id/ncrquartermaster
+	uniform =  		/obj/item/clothing/under/f13/ncr
+	accessory =     /obj/item/clothing/accessory/ncr/ssgt
+	suit = 			/obj/item/clothing/suit/armor/f13/ncrarmor/mantle
+	head = 			/obj/item/clothing/head/f13/ncr
+	glasses = 		/obj/item/clothing/glasses/sunglasses/big
+	suit_store  = /obj/item/gun/ballistic/automatic/marksman
```

Comment:
Might wanna not give them one of the best rifles in the game, that doesn't spawn on the map, and is reserved for adminspawn Legate/Colonel.

Reply:
Definitely want to consider that point. Are you purging a sergeant slot to compensate?

Reply:
did so but RenGusta (The guy above) after i removed a sergeant slot and gave them a shotgun decided to not add it and overrule compano.